tuf/api/metadata.py

```diff
@@ -351,6 +351,26 @@ def _common_fields_to_dict(self) -> Dict[str, Any]:
             "expires": self.expires.isoformat() + "Z",
         }
 
+    def is_expired(self, reference_time: datetime = None) -> bool:
+        """
+        <Purpose>
+            Checks metadata expiration against reference time.
+
+        <Arguments>
+            reference_time:
+                The time to check the metadata expiration date against.
+                A naive datetime in UTC expected.
+                If not provided, checks against the current UTC date and time.
+
+        <Returns>
+            True if the reference time is equal or
+            greater than the metadata expiration date.
+        """
```

Comment:
Thanks this helps... but it's the old style doc format which we're hoping to not use in api/

```suggestion
        """Checks metadata expiration against a reference time.

        Args:
            reference_time: Optional; The time to check expiration date against.
                A naive datetime in UTC expected.
                If not provided, checks against the current UTC date and time.

        Returns:
            True if expiration time is less than the reference time.
        """
```

I think this matches the current style (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) ... but I've never used this style yet either so maybe @MVrachev or someone more familiar can confirm if this is what we want for new code?
In any case let's try to keep it to as few lines as possible.

Reply:
Looks good, should be ok with the new style.

Reply:
Thank you all for the comments! I believe I addressed all issues and pushed a squashed commit. I promise to carefully go through the guidelines again and to do better with observing best practices and recommendations next time :)